src/modalities/__main__.py

```diff
@@ -191,24 +214,52 @@ def CMD_entry_point_pack_encoded_data(config_path: FilePath):
     help="The number of chunks to create.",
 )
 @click.option(
-    "--vocab_size",
+    "--file_existence_policy",
+    type=click.Choice([policy.value for policy in FileExistencePolicy]),
+    default=FileExistencePolicy.ERROR.value,
+    help="Policy for handling existing files.",
+)
+@click.option(
+    "--global_seed",
     type=int,
-    required=True,
-    help="The size of the vocabulary.",
+    default=None,
+    help="The global seed to use for shuffling.",
 )
 def CMD_create_shuffled_dataset_chunk(
-    input_file_list_path: Path, output_chunk_file_path: Path, chunk_id: int, num_chunks: int, vocab_size: int
+    input_file_list_path: Path,
+    input_data_root_path: Path,
+    output_chunk_file_path: Path,
+    chunk_id: int,
+    num_chunks: int,
+    file_existence_policy: FileExistencePolicy,
+    global_seed: int,
```

Comment:
```suggestion
    global_seed: Optional[int],
```
would require `from typing import Optional`.